openpdf/src/main/java/com/lowagie/text/pdf/PdfBoolean.java

```diff
@@ -50,87 +50,43 @@
 
 package com.lowagie.text.pdf;
 
-import com.lowagie.text.error_messages.MessageLocalization;
-
 /**
- * <CODE>PdfBoolean</CODE> is the boolean object represented by the keywords <VAR>true</VAR> or <VAR>false</VAR>.
- * <P>
+ * Wrapper around {@code boolean} value.
+ *
  * This object is described in the 'Portable Document Format Reference Manual version 1.7'
  * section 3.2.1 (page 52).
  *
- * @see        PdfObject
- * @see        BadPdfFormatException
+ * @see PdfBoolean#TRUE
+ * @see PdfBoolean#FALSE
+ * @see PdfBoolean#fromValue(boolean)
  */
-
-public class PdfBoolean extends PdfObject {
-    
-    // static membervariables (possible values of a boolean object)
-    public static final PdfBoolean PDFTRUE = new PdfBoolean(true);
-    public static final PdfBoolean PDFFALSE = new PdfBoolean(false);
-/** A possible value of <CODE>PdfBoolean</CODE> */
-    public static final String TRUE = "true";
-    
-/** A possible value of <CODE>PdfBoolean</CODE> */
-    public static final String FALSE = "false";
-    
-    // membervariables
-    
-/** the boolean value of this object */
-    private boolean value;
-    
-    // constructors
+public final class PdfBoolean extends PdfObject {
     
-/**
- * Constructs a <CODE>PdfBoolean</CODE>-object.
- *
- * @param        value            the value of the new <CODE>PdfObject</CODE>
- */
+    public static final PdfBoolean TRUE = new PdfBoolean(true);
+    public static final PdfBoolean FALSE = new PdfBoolean(false);
```

Comment:
the problem here is that the same constant value is used for a different type. `TRUE` and `FALSE` were of  type `String` and become `PdfBoolean`.